dali/operators/reader/loader/video_loader.cc

```diff
@@ -211,19 +212,76 @@ typename std::enable_if<!std::numeric_limits<T>::is_integer, bool>::type
         || std::abs(x-y) < std::numeric_limits<T>::min();
 }
 
-OpenFile& VideoLoader::get_or_open_file(const std::string &filename) {
+static int read_packet(void *opaque, uint8_t *buf, int buf_size) {
+  VideoFileDesc *file_data = static_cast<VideoFileDesc*>(opaque);
+
+  // open file if needed
+  if (!file_data->file_stream) {
+    file_data->file_stream = fopen(file_data->filename.c_str(), "r");
+    DALI_ENFORCE(file_data->file_stream, std::string("Could not open file ") + file_data->filename);
```

Comment:
nitpick: you could use `make_string("Could not open the file: ", file_data->filename)`

Reply:
Done